packages/node/src/utils/object.ts

```diff
@@ -12,3 +12,15 @@ export function assign<TObject, TSource1, TSource2>(
     isUndefined(srcValue) ? objValue : srcValue,
   );
 }
+
+export function camelCaseObjectKey(object) {
```

Comment:
typing
```suggestion
export function camelCaseObjectKey(object: object[]) {
```

Reply:
or rather this util function only handle single object, leave `.map` to where it is called